### ISSUE TITLE
Deliver valid OpenAPI spec to Lighthouse team for approval. 

### ISSUE THREAD

**Issue by acrollet** [department-of-veterans-affairs/va.gov-cms#6294]
## Background

As a CMS engineer, I want to set up a proof-of-concept API to be available to the public through Lighthouse, so that I can understand better if Lighthouse is a viable mechanism for exposing an API to the public.

## Acceptance Criteria

- [ ] The Lighthouse team is in receipt of a valid OpenAPI spec that describes API endpoints we are implementing as part of our POC work.

## Supporting information
Steps for setting up a sandbox API, according to [Lighthouse documentation](https://developer.va.gov/api-publishing/process):
- [x] Contact Lighthouse team with OpenAPI spec and application information
- [x] Onboarding meeting with Lighthouse team ([May 21, 2021](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/cms/accelerated_publishing/proof-of-concepts-summaries/LighthouseAPI.md))
- [ ] Preparing API for publication
- [ ] Onboard consumers

## Follow-up
Further work on setting up a Lighthouse gateway for our POC API will take place in https://github.com/department-of-veterans-affairs/va.gov-cms/issues/6888.

**Comment by indytechcook** [comment]
Added initial steps to the `Steps to create a sandbox:` part of the linked document

**Comment by indytechcook** [comment]
I updated the document with some notes regarding how the OpenAPI spec is used and updated in Lighthouse.

**Comment by jefflbrauer** [comment]
Reach out to Wyatt first, if no response still, reach out to Andrew Fichter, and if all else fails, David Mazik

**Comment by timcosgrove** [comment]
A follow-up ticket is here: https://github.com/department-of-veterans-affairs/va.gov-cms/issues/6888

This is a copy of the current ticket's state at this time. We are effectively blocked until Lighthouse accepts the OpenAPI spec that was delivered to them, and then gives us next steps to move forward with doing testing via the sandbox.

**Comment by timcosgrove** [comment]
Spec delivered: https://dsva.slack.com/archives/C021UM3QTCY/p1635962441019400?thread_ts=1635784825.013300&cid=C021UM3QTCY

Waiting on confirmation and next steps. Next steps will be in #6888

**Comment by mchelen-gov** [comment]
@timcosgrove can you attach the OpenAPI spec to this ticket or upload to Confluence?
